xbmc/windowing/gbm/drm/DRMUtils.h

```diff
@@ -62,6 +62,15 @@ class CDRMUtils
   static uint32_t FourCCWithAlpha(uint32_t fourcc);
   static uint32_t FourCCWithoutAlpha(uint32_t fourcc);
 
+  bool SupportsAsync() const { return m_supportsAsync; }
+  void SetInFenceFd(int fd) { m_inFenceFd = fd; }
+  int GetOutFenceFd()
+  {
+    int fd{-1};
+    std::swap(m_outFenceFd, fd);
+    return fd;
```

Comment:
```suggestion
    return std::exchange(m_outFenceFd, -1);
```
Also maybe rename the method to something like `TakeOutFenceFd` to emphasis that this alters the object?

Reply:
Great suggestions :+1: